### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by TopsyKretts13**

### CODE REVIEW

File: src/isIsogram.test.js [hunk 1]
```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
   });
```
Comment (IrynaKhyzhynska) [inline src/isIsogram.test.js:9]: looks good to me

File: src/isIsogram.test.js [hunk 2]
```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
...
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('1')).toBe('boolean');
   });
 
   it(`should be case sensitive`, () => {
+    expect(isIsogram(`MAtE`)).toBe(true);
+  });
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:16]: Here to check it you should have the same letter in uppercase and in lowercase (like `'Dad'` - should return `false` because `'D'` and `'d'` are the same letter for this task)

File: src/isIsogram.test.js [hunk 3]
```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
...
+  next to each other`, () => {
+    expect(isIsogram('academy')).toBe(false);
+  });
+
+  it(`should return 'false' if the same characters are 
+  next to each other`,
+  () => {
+    expect(isIsogram('pretty')).toBe(false);
+  });
 });
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:32]: Add a test for an isogram that is not an empty string

File: src/isIsogram.test.js [hunk 4]
```diff
@@ -8,12 +8,25 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('1')).toBe('boolean');
```
Comment (Oleksandra-Pasenchenko) [inline src/isIsogram.test.js:11]: Input can contain only letters, so better choose some words for the input in this test
